scripts/filter_qodana_paths.py

```
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
```

Ruff (0.14.0)

Shebang is present but file is not executable

(EXE001)